### PR TITLE
fix(console): add letter spacing for sign-in-experience title

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

fix(console): add letter spacing for sign-in-experience title

## Linear Issue Reference

* LOG-2797

## Testing

image

### CODE REVIEW

File: packages/console/src/pages/SignInExperience/components/index.module.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .title {
   color: var(--color-neutral-variant-60);
   font: var(--font-subhead-cap);
+  letter-spacing: 0.1em;
```
Comment (wangsijie) [inline packages/console/src/pages/SignInExperience/components/index.module.scss:6]: We don't use `em` in other place, is this on purpose?

Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/index.module.scss:6]: According to the design file in Figma, this value should be 10% of the font size.

Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/components/index.module.scss:6]: font size is fixed (--font-subhead-cap), so we can still use a fixed "px" here.

Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/components/index.module.scss:6]: `em` is good though, just be caution to introduce new unit. @logto-io/eng

Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/index.module.scss]: per discussion, it's ok to use `em`, and we add a subhead css mixin to make it safer.
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1654152409797829

**Review by demonzoo — APPROVED** [review]
I'm good with it

**Pull request merged**